ua/extension_object.go

```diff
@@ -10,6 +10,38 @@ import (
 	"github.com/gopcua/opcua/id"
 )
 
+// eotypes contains all known extension objects.
+var eotypes = NewTypeRegistry()
+
+// init registers known built-in extension objects.
+func init() {
+	// RegisterExtensionObject(id.AnonymousIdentityToken_Encoding_DefaultBinary, new(AnonymousIdentityToken))
```

Comment:
seems to me, there's no need for this commented out code

Reply:
Indeed ...